ffi/framebuffer_pocketbook.lua

```diff
@@ -42,7 +42,11 @@ local function _updateFull(fb, x, y, w, h, dither)
         _adjustAreaColours(fb)
     end
 
-    inkview.FullUpdate()
+    if fb.device.hasColorScreen then
+        inkview.FullUpdateHQ()
+    else
+        inkview.FullUpdate()
```

Comment:
During testing, I noticed that this does not result in a screen refresh on PocketBook devices without a color screen.

Reply:
This PR doesn't change anything for B&W Pocketbooks. Do you have full refresh turned on?

Reply:
> Do you have full refresh turned on?

Yes, of course.

> This PR doesn't change anything for B&W Pocketbooks.

Unfortunately, because of lines 45 and 46, B&W PocketBook devices now ignore line 48 for whatever reason.

Reply:
Hmm, maybe we need to explicitly set `hasColorScreen` to false. I've created https://github.com/koreader/koreader/pull/11608 to address this.

Reply:
Okay, now I'm 99% sure https://github.com/koreader/koreader-base/pull/1757 should fix it. :)